content/content/strings.md

```diff
@@ -2,6 +2,17 @@
 title: Strings
 ---
 # Strings
+In Nim, the string represents a sequence of characters. It’s a list of characters terminated by '\0'. There usage is very similar to python
```

Comment:
Nim does look very similar to python, but the unwritten style guide that I've been using here avoids making any references to other languages, so this sentence should go away.